src/main/java/core/basesyntax/Name.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Name {
```

Comment:
More precise naming is always better. Nearly anything can have name
```suggestion
public enum Shape {
```